### PR TITLE
Adds `snipeit:remove-explicit-eols` command

### PR DESCRIPTION
**Pull request by Godmartinz**
# Description
This adds `snipeit:remove-explicit-eols` . This command allows you to provide the Asset model name and then removes the explicit EOLs that are applied to all assets with that model and updating them, this will refresh the assets with the Asset Model's EOL behavior.

This also adds an `"all"` option to update all assets.

Command Options Help:
image

Updating All Assets:
image

Updating one Model:
image

Error:
image

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* PHP version:
* MySQL version
* Webserver version
* OS version

# Checklist:

- [ ] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [ ] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by snipe** [comment]
I don't see any harm in including this I guess. Normally we would just run a quick SQL command.

File: app/Console/Commands/RemoveExplicitEols.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+     * @var string
+     */
+    protected $description = 'Removes explicit EOLs on assets with selected model so they may inherit the asset model EOL';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        $assetModel= AssetModel::where('name', '=', $this->option('model_name'))->first();
```
Comment (snipe) [inline app/Console/Commands/RemoveExplicitEols.php]: How is this going to handle spaces in the model name? Should we perhaps pass it an ID instead? Maybe optionally, so we can reset all if they have a lot of models and want to handle it for all asset models.

**Review by snipe — CHANGES_REQUESTED** [review]

File: app/Console/Commands/RemoveExplicitEols.php (hunk 1 above)
Comment (Godmartinz) [reply app/Console/Commands/RemoveExplicitEols.php]: The way you enter a model name is `--model_name="Spaces in Between the quotes are fine"`
I thought about finding the model by Id, but if you put in the wrong id number you could do a lot of damage. if you misspell the model name it returns "Asset Model not Found".

**Comment by Godmartinz** [comment]
> I don't see any harm in including this I guess. Normally we would just run a quick SQL command.

the Asset Model EOL is not inherited if you clear the columns only. There is some logic in the update/save method that applies the asset model EOL if it is null.

File: app/Console/Commands/RemoveExplicitEols.php (hunk 1 above)
Comment (Godmartinz) [reply app/Console/Commands/RemoveExplicitEols.php]: i could write an all option

**Pull request merged**